tests/testthat/test-mle_finder.R

```diff
@@ -25,6 +39,20 @@ test_that("linalg and optim least-sq coincide", {
   )
   expect_true(are_all_close(
     coef(via_linalg_out), coef(via_bfgs_out),
+    abs_tol = 1e-6, rel_tol = 1e-6
+  ))
+})
+
+test_that("Newton and bfgs outputs coincide on logit model", {
```

Comment:
Hi Andrew, 
Is it proper to move the helper.R into the ./R folder here?
-Wentao

Reply:
No, helper.R is for code that's only used for tests. Since are_all_close() is now a part of the core package functionality, move it somewhere into the package. For now you can put it right before `mle_Newton_logit()`. And then the tests can access it still since it'll get loaded with the other functions.

Reply:
making a note that due to the R version weirdness that this passes in R4.2, so this deduction won't be counted.